modules/custom/d_commerce/modules/d_commerce_product/src/Entity/ProductVariation.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Drupal\d_commerce_product\Entity;
+
+use Drupal\commerce_product\Entity\ProductVariation as CommerceProductVariation;
+
+/**
+ * Provides an override for commerce_product product variation entity class.
+ *
+ * @package Drupal\d_commerce_product\Entity
+ */
+class ProductVariation extends CommerceProductVariation {
```

Comment:
Taka luźna myśl - czy jak nazwiemy tę klasę `ProductVariation` zamiast np. `DrooplerProductVariation`, to czy nie będzie myliło się z `Drupal\commerce_product\Entity\ProductVariation`?

Reply:
Nic nie sugeruję, zastanawiam się tylko jak to robią inni

Reply:
Ogólnie po to jest namespace. Bardzo dużo klas się pokrywa nazwami ze sobą ogólnie core/contrib, z vendorów też, często w Stormie musze patrzeć skąd faktycznie to pochodzi.
Ja bym chyba zostawił jako ProductVariation...